### PR TITLE
Feature/setup eslint in precommit

### PR DESCRIPTION
**Pull request by abolkog**
This commit: 
* Add eslint to git pre-commit hook. This will make us unable to commit our code if we have any linter errors. 
* Remove/comment out all unused constants and console.logs

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -18,23 +18,16 @@ const Footer = () => {
...
-                </li>
+                <li></li>
                 <li></li>
               </ul>
             </div>
           </div>
           <div className="flex w-full md:w-1/3">
             <div className="px-8">
-              <h3 className="font-bold font-bold text-gray-100">Language</h3>
+              <h3 className="ont-bold text-gray-100">Language</h3>
```
Comment (kosenchiha) [inline src/components/Footer.js:30]: It looks like "f" is missing :)

File: src/components/Signin/SigninForm.js [hunk 2]
```diff
@@ -137,7 +146,7 @@ const SigninForm = () => {
 
         <div className="text-center">
           <p className="text-grey-dark text-sm">
-            Don't have an account?{' '}
+            {`Don't have an account?`}
             <a href="/signup" className="no-underline text-blue font-bold">
```
Comment (kosenchiha) [inline src/components/Signin/SigninForm.js:150]: Just out of curiosity, I've noticed sometimes you use <a> and sometimes <Link> is there any rule for these tags?

Comment (abolkog) [reply src/components/Signin/SigninForm.js:150]: I think the gatsby Link tag should be used for internal components while the other tag should be used for external links

File: src/components/Footer.js (hunk 1 above)
Comment (abolkog) [reply src/components/Footer.js:30]: nice catch

**Pull request merged**